ui/desktop/tests/e2e/targets.spec.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+const { test } = require('./fixtures/baseTest');
+const { expect } = require('@playwright/test');
+
+test.describe('Targets test', async () => {
+  test('Connects to a target and ends session', async ({ authedPage }) => {
```

Comment:
Should we add a test for connecting from the targets route as well?

Reply:
Yup probably. This wasn't to be an exhaustive list of tests, just to setup the test setups and fix what the previous existing tests were trying to do.

We'll need to add a lot more tests in the future but that's not the scope of this PR.